src/sentry/receivers/releases.py

```diff
@@ -103,7 +156,7 @@ def resolved_in_pull_request(instance, created, **kwargs):
     )
     for link in group_links:
         if link.group_id not in group_ids:
-            link.delete()
+            remove_resolved_link(group, link)
```

Comment:
This function only takes one argument.

Reply:
I *think* the linter would have caught this (it is in VS Code), but im still not sure why its not running it locally for me.